posts/2024/2024_05_31_Inigo_week_1.rst

```diff
@@ -0,0 +1,24 @@
+First Week into GSoC 2024: Building the AutoEncoder, writing the training loop
+==============================================================================
+
+.. post:: May 31 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
+I finished becoming familiar with the TensorFlow + Keras basics and I wrote the training loop and a couple of scripts for instantiating and training the AutoEncoder.
+Data loading was also addressed and I am able to load the data from the FiberCup dataset in .trk format using `NiBabel <https://nipy.org/nibabel/>`__, transform it into NumPy arrays, and feed it into the network.
```

Comment:
Again, a single underscore is enough in links: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Reply:
fixed, thanks :)